src/main/java/com/learn/java/inheritance/AbstractRunner.java

```diff
@@ -0,0 +1,7 @@
+package com.learn.java.inheritance;
+
+public abstract class AbstractRunner implements Runner {
+    public void run(){
+        System.out.println(2);
```

Comment:
modify the run method in order to receive an argument runner and print "the _runner_ runs"